webapp/Connector/src/component/AdvancedOption.js

```diff
@@ -410,7 +443,7 @@ Ext.define('Connector.component.AdvancedOptionTime', {
 
     constructor : function(config) {
         if (config.measure == undefined || config.measure.$className !== 'Connector.model.Measure') {
-            console.error('Advanced option scale field must be defined using a Measure record.');
+            console.error('Advanced option time field must be defined using a Measure record.');
```

Comment:
Good catch. I fixed by renaming this component to Connector.component.AdvancedOptionTimeAlignedBy.